src/utils/math/kalman_multivariate.cpp

```diff
@@ -39,21 +39,44 @@ namespace hyped {
 
             void KalmanMultivariate::setDynamicsModel(MatrixXf& A, MatrixXf& Q)
             {
+                if (A.cols()!= n_ || A.rows()!= n_ || Q.cols()!=n_ ||  Q.rows()!=n_ )
```

Comment:
Try reformatting these conditional statements so they're more aligned with the style guide. Mainly the curly braces should be on the same line. [More info here](https://hyp-ed.github.io/styleguide/#Conditionals)

Reply:
I think it should be more or less fixed now, linter is not working when I am trying to push on this branch which is weird

Reply:
Also, I already have a version of tests for these exceptions, I am waiting for these changes to be on develop to paste them in the new branch of Kalman tests, right now I have two new ones

Reply:
Is the linter not working at all? Does it come up with anything linter-related or just doesn't run?